### ISSUE TITLE
It's possible to generate feedback docs with no error attached

### ISSUE THREAD

**Issue by SCdF** [medic/medic#4483]
We need to look into how feedback docs get generated to make sure that the logic still holds up. It's possible to generate feedback docs that are just 10 non-erroring log messages and no error.

It may be that it broke recently for some reason.

**Comment by garethbowen** [comment]
This happens by design if you click the "report bug" button. @SCdF Is it possible this happened? Did _you_ create the doc or was it just something you saw in a db somewhere?

I'm tempted to close this until we have an example we can investigate.

**Comment by SCdF** [comment]
@garethbowen I didn't create it no. It was either:
 - Not generated from a manual report bug action, and a bug in our error generation
 - Generated from a manual report bug action, but the user just clicked send without typing anything and we allow it.

This is because the entire document was just 10 lines of debug output and no other information (eg completely non-actionable).

**Comment by garethbowen** [comment]
I've just tested locally and we _do_ allow reporting bugs without typing anything. The doc I created is...

```json
{
  "_id": "2a945fc16f287378d002ac7f1500221e",
  "_rev": "1-247ac7dba4964f75c5ca7160d8b4a03d",
  "meta": {
    "time": "2018-11-05T20:28:38.065Z",
    "user": {
      "name": "admin",
      "roles": [
        "_admin"
      ]
    },
    "url": "http://localhost:5988/medic/_design/medic/_rewrite/#/contacts/34ec74c8-d4e4-4109-a44b-8a3d83d880eb",
    "app": "medic",
    "version": "3.2.0"
  },
  "log": [
    {
      "level": "log",
      "arguments": "{\"0\":\"Data node: /*/meta/deprecatedID with null-based index: undefined not found. Ignored.\"}"
    }
  ],
  "type": "feedback"
}
```

I'm thinking we should mark when the feedback doc was manually generated vs automatically generated. Also we could force users to type a message when manually generating feedback.

@SCdF What do you think?

**Comment by SCdF** [comment]
@garethbowen both of those sound like good ideas to me.

(it's especially terrible that you didn't even get a field with an empty string in it)

**Comment by garethbowen** [comment]
Ok, do those things I suggest above.

**Comment by ngaruko** [comment]
Old ticket. LGTM and already merged I guessed in PR #5193  @wtekeu ?
![image](https://user-images.githubusercontent.com/6979995/55772210-6d54ed80-5adf-11e9-85d4-9bc9cf64ea14.png)

**Comment by wtekeu** [comment]
Yes @ngaruko already merged
